mistletoe/block_token.py

```diff
@@ -18,7 +18,7 @@
 """
 Tokens to be included in the parsing process, in the order specified.
 """
-__all__ = ['BlockCode', 'Heading', 'Quote', 'CodeFence', 'ThematicBreak',
+__all__ = ['HTMLAttributes', 'BlockCode', 'Heading', 'Quote', 'CodeFence', 'ThematicBreak',
```

Comment:
You don't need to add the HTMLAttributes class here: it's added automatically when you pass it to the HTMLRenderer constructor from the HTMLAttributesRenderer constructor.

Reply:
Yeah, I think this line is already linked / discussed in an older thread talking about performance of existing parsing and conditionality of this feature.

Reply:
See (cd6ff447f92ea39b5972ef08bb81cdd21a0dc1ff)